docs/api/other-api/adapter.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "@react-router/{adapter}"
+---
+
+# Server Adapters
+
+## Official Adapters
+
+Idiomatic React Router apps can generally be deployed anywhere because React Router adapts the server's request/response to the [Web Fetch API][web-fetch-api]. It does this through adapters. We maintain a few adapters:
+
+- `@react-router/architect`
+- `@react-router/cloudflare`
+- `@react-router/express`
+
+These adapters are imported into your server's entry and are not used inside your React Router app itself.
+
+If you initialized your app with `npx create-react-router@latest` with something other than the built-in [React Router App Server][rr-serve] (`@react-router/serve`), you will note a `server/index.js` file that imports and uses one of these adapters.
+
+<docs-info>If you're using the built-in React Router App Server, you don't interact with this API</docs-info>
+
+Each adapter has the same API. In the future we may have helpers specific to the platform you're deploying to.
+
+## `@react-router/express`
+
+[Reference Documentation ↗](https://api.reactrouter.com/v7/modules/_react_router_express.html)
```

Comment:
I think this example might be a bit confusing, it leaves out the part where you import `createRequestHandler` from `@react-router/express` and set it up

Reply:
oops - I must have copied in the wrong thing - the correct express example got left at the bottom of this doc by accident